algorithms/scaling/scaling_utilities.py

```diff
@@ -6,7 +6,9 @@
 import logging
 from math import acos
 
-import scitbx
+import numpy as np
+from scipy.spatial.transform import Rotation
```

Comment:
FWIW scipy also has [vector.ReferenceFrame](https://docs.sympy.org/latest/modules/physics/vector/api/classes.html), which is neat because yone just define reference frames with respect to one another and then call the vector in the wanted reference frame, and it does quaternion coordinate transform in the background.

Reply:
That appears to be `SymPy` (which I hadn't come across before) rather than `SciPy`...

Reply:
Ah fair